11LessonEleven/in-class-project/index.html

```diff
@@ -4,7 +4,31 @@
   <title>Intel</title>
```

Comment:
your `@media (max-width: 767px)` it's not working because of two things:
* it's commented
* You are missing the `meta viewport` in your `<head>...</head>` element

this is the `meta viewport` you are missing:
<meta name="viewport" content="width=device-width, initial-scale=1">